db/migrate/20151021174044_add_tenant_default_group.rb

```diff
@@ -1,5 +1,8 @@
 class AddTenantDefaultGroup < ActiveRecord::Migration
+  class Tenant < ActiveRecord::Base; end
+
   def change
     add_column :tenants, :default_miq_group_id, :bigint
+    Tenant.reset_column_information
```

Comment:
Does creating the stub class on line of this file negate the need for this reset?